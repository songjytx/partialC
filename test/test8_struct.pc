struct Test{
	int a;
	float b;
	bool c;
};

void main(){
	Test x;
	x.a = 1;
	x.b = 4.5;
	x.c = true;

	printi(x.a);
	printf(x.b);
	
}
