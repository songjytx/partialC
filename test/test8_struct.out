1
4.500000
'success!'
